### PR TITLE
feat(vscode): Organize Azure workspace commands into submenu

### PR DESCRIPTION
**Pull request by alexweininger**
Hello from the Azure extension team!

We are organizing the workspace `view/title` commands into submenus so that there is less clutter in the workspace menu. See https://github.com/microsoft/vscode-azureresourcegroups/blob/main/api/README.md#viewtitle-commands for more details. 

Here's what that looks like for Logic Apps:
image

We've done the same for all of our extensions. Here's what it looks like for Azure Functions:
image

AB#24363706

### CODE REVIEW

File: apps/vs-code-designer/src/package.json [hunk 1]
```diff
@@ -606,10 +601,6 @@
           "command": "azureLogicAppsStandard.openDesigner",
           "when": "resourceFilename==workflow.json"
         },
-        {
```
Comment (ccastrotrejo) [inline apps/vs-code-designer/src/package.json:609]: You are removing this because you don't have the latest from main

**Review by ccastrotrejo — CHANGES_REQUESTED** [review]
Overall, its good. Please just update your branch with latest of main.

File: apps/vs-code-designer/src/package.json (hunk 1 above)
Comment (alexweininger) [reply apps/vs-code-designer/src/package.json:609]: Merged with main

File: CHANGELOG.md
Comment (alexweininger) [inline CHANGELOG.md:1]: Not sure what happened here. Let me know if you want me to revert.

Comment (ccastrotrejo) [reply CHANGELOG.md:1]: If you can yes, please

File: apps/vs-code-designer/src/package.json (hunk 1 above)
Comment (ccastrotrejo) [inline apps/vs-code-designer/src/package.json:609]: It seems that you still doesn't have these changes

**Review by ccastrotrejo — APPROVED** [review]

**Comment by ccastrotrejo** [comment]
@hartra344 FYI

**Pull request merged**